src/PhpSpec/CodeAnalysis/AccessInspectorInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Phpspec\CodeAnalysis;
+
+interface AccessInspectorInterface
```

Comment:
IMO, the implementation should not be named ``AccessInspector``. Otherwise it is a pain to change the convention later. We should already name the class as if the name ``AccessInspector`` was reserved by the interface itself